package.json

```diff
@@ -24,7 +24,7 @@
     "lint": "standard",
     "prebuild": "npm run lint",
     "postpublish": "npm run deploy",
-    "posttest": "codecov",
+    "posttest": "[ -z \"$CI\" ] || codecov",
```

Comment:
True. Unfortunately there are no official collaborators besides myself so it doesn't pose any problems currently.